### ISSUE TITLE
Pachtop uses too many resources so measurement is garbled

### ISSUE THREAD

**Issue by martin-pilka-tfs** [pacholoamit/pachtop#6]
From `htop`:
![Selection_042](https://user-images.githubusercontent.com/110905436/204790268-d7169cf2-3d42-46dc-8e3e-a8b31b1fcbeb.png)

Running without `sudo` (fan is very loud):
```
mpilka@e15:~/Downloads$ pachtop 
libGL error: failed to open /dev/dri/card0: Permission denied
libGL error: failed to open /dev/dri/card0: Permission denied
libGL error: failed to load driver: iris
```
![Workspace 2_001](https://user-images.githubusercontent.com/110905436/204790438-37367154-e722-4070-ab5f-5b468258c9fa.png)

Running with `sudo` (fan is OK):
```
mpilka@e15:~/Downloads$ sudo pachtop
```
![Workspace 2_002](https://user-images.githubusercontent.com/110905436/204790732-9fa0c3ee-161f-41cd-9dea-9212b7ecbbde.png)

**Comment by pacholoamit** [comment]
Hello! I'll look into this when I have time but Pachtop is still not stable as I'm currently experimenting on additional features such as persisting metrics into an sqlite db so the user would have persistent records of their system's metrics over time.

**Comment by pacholoamit** [comment]
Performance issues should have been addressed on version 0.4.0
